relay-cabi/src/processing.rs

```diff
@@ -220,3 +221,22 @@ pub unsafe extern "C" fn relay_parse_release(value: *const RelayStr) -> RelayStr
     let release = sentry_release_parser::Release::parse((*value).as_str())?;
     RelayStr::from_string(serde_json::to_string(&release)?)
 }
+
+/// Validate a dynamic rule condition.
+/// We only validate conditions in Relay since conditions are the only part of the rules that are
+/// volatile and need constant updating to keep in sync between Relay and Sentry
+#[no_mangle]
+#[relay_ffi::catch_unwind]
+pub unsafe extern "C" fn relay_validate_dynamic_rule_condition(value: *const RelayStr) -> RelayStr {
+    let ret_val = match serde_json::from_str::<RuleCondition>((*value).as_str()) {
+        Ok(condition) => {
+            if condition.supported() {
+                "".to_string()
+            } else {
+                "invalid condition".to_string()
```

Comment:
This should probably be `unsupported condition`?

Reply:
yeah, that's a better error, changed.